src/services/ubi/community.ts

```diff
@@ -727,12 +729,256 @@ export default class CommunityService {
+                        ),
+                    },
+                },
+            ],
+            where: {
+                id: communityId,
+            },
+            group: ['Community.id'],
+            raw: true,
+        })) as any;
+
+        const communityRemovedBeneficiaryActivity: {
+            beneficiaries: string;
+        } = (await models.community.findOne({
+            attributes: [
+                [fn('count', col('beneficiaries.address')), 'beneficiaries'],
+            ],
+            include: [
+                {
+                    model: models.beneficiary,
+                    as: 'beneficiaries',
+                    attributes: [],
+                    required: false,
+                    where: {
+                        updatedAt: where(
+                            fn('date', col('beneficiaries."updatedAt"')),
+                            '=',
+                            todayDate
+                        ),
+                        active: false,
+                    },
+                },
+            ],
+            where: {
+                id: communityId,
+            },
+            group: ['Community.id'],
+            raw: true,
+        })) as any;
+
+        const communityNewManagerActivity: {
+            managers: string;
+        } = (await models.community.findOne({
+            attributes: [[fn('count', col('managers.address')), 'managers']],
+            include: [
+                {
+                    model: models.manager,
+                    as: 'managers',
+                    attributes: [],
+                    required: false,
+                    where: {
+                        createdAt: where(
+                            fn('date', col('managers."createdAt"')),
+                            '=',
+                            todayDate
+                        ),
+                    },
+                },
+            ],
+            where: {
+                id: communityId,
+            },
+            group: ['Community.id'],
+            raw: true,
+        })) as any;
+
+        const communityRemovedManagerActivity: {
+            managers: string;
+        } = (await models.community.findOne({
+            attributes: [[fn('count', col('managers.address')), 'managers']],
+            include: [
+                {
+                    model: models.manager,
+                    as: 'managers',
+                    attributes: [],
+                    required: false,
+                    where: {
+                        updatedAt: where(
+                            fn('date', col('managers."updatedAt"')),
+                            '=',
+                            todayDate
+                        ),
+                        active: false,
+                    },
+                },
+            ],
+            where: {
+                id: communityId,
+            },
+            group: ['Community.id'],
+            raw: true,
+        })) as any;
```

Comment:
Now that you've got to this point, simplify.
No need for groups, do nested includes.

My suggestion, get the community contract address, and then do each one of these queries, individually, like `select sum(amount) from inflow where today & community=x`

Reply:
getState refactored